sdk/arm/network/2020-03-01/armnetwork/availableendpointservices.go.601121107

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why is this file name like this? `sdk/arm/network/2020-03-01/armnetwork/availableendpointservices.go.601121107 `

Reply:
Weird, seems to be a duplicate of `availableendpointservices.go`, will remove.